.github/workflows/ci.yml

```diff
@@ -12,8 +12,9 @@ jobs:
     strategy:
       matrix:
         python-version: ['3.8']
-        django-version: ['django22']
-        toxenv: ['django22', 'quality', 'docs', 'pii-annotations']
+        toxenv: [
+          'django22-celery44', 'django22-celery50', 'quality', 'docs', 'pii-annotations',
```

Comment:
There are still two conditions below checking against "django22", those should probably be updated to use one of the new `toxenv` values.